barretenberg/cpp/CMakePresets.json

```diff
@@ -89,6 +89,7 @@
       "displayName": "Build with GCC",
       "description": "Build with globally installed GCC",
       "inherits": "default",
+      "binaryDir": "build-gcc",
```

Comment:
This just makes it easier to use `./scripts/docker_interactive.sh` then `cmake --preset gcc` without moving the native build dir